src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5435,6 +5444,15 @@ CalcHeatBalanceInsideSurf( Optional_int_const ZoneToResimulate ) // if passed in
 		}
 	}
 
+	// Calculate heat extract due to additional heat flux source term as the surface boundary condition 
+	for ( SurfNum = 1; SurfNum <= TotSurfaces; ++SurfNum ) {
+		if( Surface( SurfNum ).HasAdditionalHeatSourceTerm ) {
```

Comment:
With my comment above, this double layer if block would reduce to just a single `if HasInsideHeatSourceTerm` block.